starter-core/src/test/groovy/io/micronaut/starter/feature/buildless/BuildlessSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package io.micronaut.starter.feature.buildless
+
+import io.micronaut.starter.ApplicationContextSpec
+import io.micronaut.starter.application.ApplicationType
+import io.micronaut.starter.build.dependencies.CoordinateResolver
+import io.micronaut.starter.feature.Category
+import io.micronaut.starter.fixture.CommandOutputFixture
+import io.micronaut.starter.options.BuildTool
+import io.micronaut.starter.options.Language
+import io.micronaut.starter.options.Options
+
+class BuildlessSpec extends ApplicationContextSpec implements CommandOutputFixture {
+    @Override
+    Map<String, Object> getConfiguration() {
+        ["spec.name": "BuildlessSpec"]
+    }
+
+    void 'test README.md with buildless feature contains links to docs'() {
+        when:
+        def output = generate([Buildless.NAME])
+        def readme = output["README.md"]
+
+        then:
+        readme.contains("[https://docs.less.build/](https://docs.less.build/)")
+    }
+
+    void "feature buildless properties validation"() {
+        given:
+        def feature = beanContext.getBean(Buildless)
+
+        expect:
+        for (applicationType in ApplicationType.values()) {
+            assert feature.supports(applicationType)
+        }
+
+        feature.category == Category.DEV_TOOLS
+        feature.name == Buildless.NAME
+    }
+
+    void "buildless configured correctly for #buildTool"() {
```

Comment:
```suggestion
    void "feature buildless is unsupported with maven and #language"(Language language) {
        when:
        generate(ApplicationType.DEFAULT, new Options(language, BuildTool.MAVEN), [Buildless.NAME])

        then:
        def e = thrown(IllegalArgumentException)
        e.message == 'Feature only supported by Gradle'

        where:
        language << Language.values()
    }

    void "buildless configured correctly for #buildTool"() {
```

Reply:
Did we not add the test for maven?

Reply:
@timyates I adopted 'GradleFeature' to make this Gradle-only for now, as our Maven integration is in beta. I intend to return with a new PR for Maven after the U.S. holiday.